### ISSUE TITLE
Windows formatted files

### ISSUE THREAD

**Issue by chiras** [molbiodiv/bcdatabaser_web#15]
Windows formatted files yield errors, because of the line breaks: see. eg. https://bcdatabaser.molecular.eco/job_details?id=7

**Comment by iimog** [comment]
Good catch. This is actually an issue that exists upstream in the command line script. So I will fix it there :-)
